module/localisation/RobotLocalisation/src/RobotLocalisation.cpp

```diff
@@ -43,8 +46,10 @@ namespace module::localisation {
 
     using LocalisationRobot  = message::localisation::Robot;
     using LocalisationRobots = message::localisation::Robots;
+    using LocalisationField  = message::localisation::Field;
```

Comment:
```suggestion
    using message::localisation::Field;
```
since there isn't anything else called Field, you don't need to give it an alias